### ISSUE TITLE
Include grideditors json in package.manifest

### ISSUE THREAD

**Issue by bjarnef** [umco/umbraco-doc-type-grid-editor#106]
In the latest versions of Umbraco (not sure from which version this was supported) it is possible to include custom grid editors from `package.manifest` files in `/App_Plugins` folder, e.g.

```
{
  "gridEditors": [
    {
      "name": "Doc Type",
      "alias": "docType",
      "view": "/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.html",
      "render": "/App_Plugins/DocTypeGridEditor/Render/DocTypeGridEditor.cshtml",
      "icon": "icon-item-arrangement",
      "config": {
        "allowedDocTypes": [],
        "nameTemplate": "",
        "enablePreview": true,
        "viewPath": "/Views/Partials/Grid/Editors/DocTypeGridEditor/",
        "previewViewPath": "/Views/Partials/Grid/Editors/DocTypeGridEditor/Previews/",
        "previewCssFilePath": "",
        "previewJsFilePath": ""
      }
    }
  ],
  "javascript": [
    "~/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.resources.js",
    "~/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.services.js",
    "~/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.controllers.js",
    "~/App_Plugins/DocTypeGridEditor/Js/doctypegrideditor.directives.js"
  ],
  "css": [
    "~/App_Plugins/DocTypeGridEditor/Css/doctypegrideditor.css"
  ]
}
```

However LeBlender only list grid editors in `/Config/grid.editors.config` json and not any inside `App_Plugins` folder, so it might be better to keep it in `/Config/grid.editors.config` json?

**Comment by leekelleher** [comment]
Thanks @bjarnef! If we could find out which is the minimum version of Umbraco that supported this, we can look to add it in.  DTGE currently supports v7.4.0+, (we're looking to bump that up to v7.7.0 for the next major version).

**Comment by leekelleher** [comment]
Closing, implemented in commit 4ce8140ca5047c8a51d6727a1602b5226f040e44 (for future v1.0.0)
